### PR TITLE
Saile

### PR DESCRIPTION
**Pull request by sailesh3571**
pulll

### CODE REVIEW

File: _pins/githubschool.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubschool:fupronix
+latitude:-33.887628
+longitude:18.565221
+---
```
Comment (hectorsector) [inline _pins/githubschool.json]: @sailesh3571 you'll need to remove this file altogether

File: _pins/sail.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: sail
```
Comment (hectorsector) [inline _pins/sail.json]: @sailesh3571 you may want to update `githubHandle` so it contains your actual username.

File: _pins/sail.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Latitude:37
+Longitude:37
```
Comment (hectorsector) [inline _pins/sail.json]: These should be case sensitive (`latitude` and `longitude`) and should have a space after `:`.

**Comment by beardofedu** [comment]
@sailesh3571 thank you for checking out our [on-demand classes](https://services.github.com/on-demand/). Since this file is not valid per the examples in the course, I am going to go ahead and close this Pull Request. If you would like to complete the activity, please feel free to re-open this Pull Request or start again. If you have any questions, please let us know!

**Pull request closed without merging**